generators/generator-adaptive-bot/generators/app/templates/dotnet/webapp/Startup.cs

```diff
@@ -39,5 +43,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                    endpoints.MapControllers();
                });
         }
+
+        private static BotAdapter RegisterCoreBotAdapter(IServiceCollection services)
```

Comment:
This cruft should go away as soon as we add a new RC build or we'll do a preview patch right after Rc0 ships.